ramp-database/ramp_database/tests/data/ramp_config_boston_housing.yml

```diff
@@ -3,12 +3,11 @@ ramp:
     event_name: boston_housing_test
     event_title: "Boston housing event"
     event_is_public: true
-    deployment_dir: /tmp/databoard_test
-    data_dir: ramp-data/boston_housing
-    kit_dir: ramp-kits/boston_housing
-    submissions_dir: submissions
-    predictions_dir: preds
-    logs_dir: log
+    data_dir: /tmp/databoard_test/ramp-data/boston_housing
+    kit_dir: /tmp/databoard_test/ramp-kits/boston_housing
+    submissions_dir: /tmp/databoard_test/submissions
+    predictions_dir: /tmp/databoard_test/preds
+    logs_dir: /tmp/databoard_test/log
```

Comment:
question: if you have multiple events, are they all stored in the same directory? (it's not stored in a subdir of this logs_dir path?)

Reply:
This is used by the dispatcher. So if you have a config file by event, you can change prediction and log directories